react/features/base/ui/components/native/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Button } from './Button';
```

Comment:
Should we add the rest of the  components here?

Reply:
We can do that yes. I don't see a point in not doing it. 
Maybe in another PR because it may take alot of import updates.